### ISSUE TITLE
Invalid test result from IE when % font size is used

### ISSUE THREAD

**Issue by nowssuming** [squizlabs/HTML_CodeSniffer#184]
When % font-size(e.g. font-size: 75%) is used in css div class, running accessibility testing on Chrome and Firefox returns correct result;however, when running test on IE, the font is considered somehow larger size of font and failed on "WCAG2AA.Principle1.Guideline1_4.1_4_3.G145.Fail" criteria while actual font is definitely smaller than 14pt.  

Result is produced from Selenium test that Consumes http://repo2/repository/com.github.squizlabs/HTMLCS-2016-04-28.js. Also manually verified using HTML_CODESNIFFER book mark tool provided on the Squiz website.

**Comment by ironikart** [comment]
This may be a duplicate of issue #155. Can you confirm you get the issue using the code on the master branch? This is a candidate for release and should be out soon.

**Comment by nowssuming** [comment]
It tried with HTMLCS checker from the master branch. The problem still exists.
